packages/web-components/src/components/ic-chip/ic-chip.tsx

```diff
@@ -40,6 +41,12 @@ export class Chip {
    */
   @Prop() appearance?: IcChipAppearance;
 
+  /**
+   * The custom chip colour. This prop will be applied to the chip component if `dismissible=false`.
```

Comment:
```suggestion
   * The custom chip colour. This prop will be applied to the chip component if `dismissible` is set to `false`.
```

just to match other places we mention props set to values